python/paddle/fluid/dygraph/amp/auto_cast.py

```diff
@@ -179,6 +292,129 @@ def amp_guard(enable=True, custom_white_list=None, custom_black_list=None):
         yield
     finally:
         if tracer:
-            tracer._enable_autocast = original_enable
+            tracer._enable_amp_l1 = original_enable_amp_l1
+            tracer._enable_amp_l2 = original_enable_amp_l2
```

Comment:
change `tracer.enebale_amp` to `tracer.amp_level`

Reply:
Done，thanks！